pkg/networkutils/network_test.go

```diff
@@ -44,6 +44,7 @@ const (
 	testeniIP        = "10.10.10.20"
 	testeniMAC       = "01:23:45:67:89:ab"
 	testeniSubnet    = "10.10.0.0/16"
+	testMTU          = 9001
```

Comment:
same here, can u also add a comment that 9001 needs to be defined in network.go. thanks